### ISSUE TITLE
White screen with Firefox

### ISSUE THREAD

**Issue by ClemPera** [corbindavenport/mini-google-tasks#4]
Hello,

I have the addon of mini-google-task (Firefox nightly v.64.0a1)
And I have a white rectangle but nothing in it (see the screenshot : )

![screenshot_20181020_105938](https://user-images.githubusercontent.com/18120892/47253781-687c7e80-d458-11e8-881f-59324db362a4.png)

There is something to do for avoid that ?

**Comment by corbindavenport** [comment]
Are you logged into a Google account?

**Comment by armeclemes** [comment]
Yes, I'm logged into my google account

**Comment by armeclemes** [comment]
Hey, 

So, do you found something to do ?

**Comment by armeclemes** [comment]
Hey, 

So, do you found something to do ?

**Comment by corbindavenport** [comment]
I still can't duplicate the bug.

**Comment by corbindavenport** [comment]
Okay, I was finally able to duplicate this. It's due to Firefox's content blocker:

```
The resource at “https://mail.google.com/tasks/ig” was blocked because content blocking is enabled.[Learn More] sidebar.html
```

I'll have to see if there is a way to whitelist extension-linked pages.

**Comment by corbindavenport** [comment]
I don't think this is something I can fix, I've reported a bug with Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1526571
